### ISSUE TITLE
Feature Request - Ingest polygons from Marine Regions as Fun and Flexible Features 

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#6930]
**Is your feature request related to a problem? Please describe.**
_A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]_

Find everything from "Atlantic Ocean"

**Describe what you're trying to accomplish** 
_A clear and concise overview of the goals; why are you asking for this?_

Enhance search for marine geography without adding to higher geography authorities.

**Describe the solution you'd like**
_How might we accomplish your goals?_

Ingest polygons from [Marine Regions](https://www.marineregions.org/) as Fun and Flexible Features

**Describe alternatives you've considered**
_A clear and concise description of any alternative solutions or features you've considered._

There is no good alternative as right now the higher geography for the Atlantic is split between North Atlantic and South Atlantic

**Additional context**
_Add any other context or screenshots about the feature request here._

@ArctosDB/geo-group 

**Priority**
_Please assign a priority-label. Unprioritized issues gets sent into a black hole of despair._

**Comment by Jegelewicz** [comment]
I think that maybe we should make narrower requests since marine regions is an aggregation.

**Comment by dustymc** [comment]
Blocked by https://github.com/ArctosDB/arctos/issues/6521 (and perhaps a general lack of performance, although I can't quite correlate that with data quantity).

>narrower requests 

I'll need a shapefile (or something else that my limited tools can handle) if I get past the immediate obstacles, that'll probably scope the request(s).

This would also be some unknown amount of disk (for data and indexes-n-such), take more backup, need more memory to avoid swap, etc. - it's a potentially significant long-term cost, that should be somehow addressed up front. (The cost seems well worth paying from here, the benefits would be pretty awesome - if this could be made performant, anyway - but I'm not signing the checks!)

**Comment by Jegelewicz** [comment]
> The cost seems well worth paying from here, the benefits would be pretty awesome - if this could be made performant, anyway - but I'm not signing the checks!

It is the cost that makes me think we should not import marine regions wholesale, but only things the member institutions actually need.

**Comment by mkoo** [comment]
Working on the FFF= POLYGONS for this.

Will definitely be part of BerkeleyMapper's intersection tool, too

**Comment by dustymc** [comment]
>cost that makes me think we should not import marine regions wholesale,

Well one way to pay that might be to loop marinregions into https://github.com/ArctosDB/arctos/issues/5597 (eg some big collaborative grant) - maybe they'd like to not have to worry about infrastructure, and I'm pretty sure TACC has a few billion GPU cycles laying around getting bored.... 

https://www.marineregions.org/about.php (maybe they're already drowning in €s....)

That'd also mean BM wouldn't need to do all the stuff Arctos has already done.

**Comment by genevieve-anderegg** [comment]
> It is the cost that makes me think we should not import marine regions wholesale, but only things the member institutions actually need.

Here are a few to start with that are quite relevant to DMNS:Inv data: 

https://www.marineregions.org/gazetteer.php?p=details&id=7579 - Great Barrier Reef (n=377)

https://www.marineregions.org/gazetteer.php?p=details&id=7587 - Peru-Chile Trench (used in https://arctos.database.museum/guid/DMNS:Inv:26912

https://www.marineregions.org/gazetteer.php?p=details&id=8453 - United States Exclusive Economic Zone (Hawaii) (several hundred)
